viewing_party/party.py

```diff
@@ -0,0 +1,162 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        new_movie = {}
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+    else: 
+        new_movie = None
+    return new_movie
+    
+def add_to_watched(user_data, movie):
+    if movie:
+        user_data["watched"].append(movie)
+    else:
+        user_data["watched"]    
+    return user_data
```

Comment:
```suggestion
    if movie:
        user_data["watched"].append(movie)

    return user_data
```

You can omit the else clause here as there's no need to explicitly tell Python to do  _nothing_ (python will assume as much). A way of reading this block of code is:

> If and only if there is a movie passed into this function, THEN append it to the 'watched' list. Regardless of adding a movie or not, return `user_data`.